### PR TITLE
Update Project:

### PR DESCRIPTION
**Pull request by ra1ndr0pGit**
- Add Google login

Issues:
- User information has not been optimized

### CODE REVIEW

File: src/main/java/com/filmbooking/controller/customer/account/GoogleLoginController.java
Comment (nphuonha2101) [inline src/main/java/com/filmbooking/controller/customer/account/GoogleLoginController.java:1]: Need to comment to explain code

File: src/main/java/com/filmbooking/model/GoogleUserInfo.java
Comment (nphuonha2101) [inline src/main/java/com/filmbooking/model/GoogleUserInfo.java:1]: Class's attributes must be wrote in camel case

**Review by nphuonha2101 — APPROVED** [review]
Oke

**Review by moclan01 — COMMENTED** [review]
good

**Pull request merged**